modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/bookmark/BookmarkPage.kt

```diff
@@ -126,10 +126,11 @@ private fun Content(isNewBookmark: Boolean, title: TextFieldValue, tintColor: Co
         }
 
         Spacer(Modifier.weight(1f))
-
+        val isTitleEnable = title.text.isBlank()
```

Comment:
Can we rename `isTitleEnable` to `isTitleBlank`?

Reply:
okay let me do that

Reply:
@ashiagr I pushed a commit